### ISSUE TITLE
[Feature Request] Adjustable autowidth 

### ISSUE THREAD

**Issue by rapatel0** [deathau/sliding-panes-obsidian#74]
I would like to use the auto width function bu

**Comment by rapatel0** [comment]
Potential settings:

`Auto Leaf Width (Mobile)`: [0%-100%] 
`Auto Leaf Width (Desktop): [0%-100%]

**Comment by haroldsnyers** [comment]
Yeah, would be great to have this, doesn't work well on mobile. the edit-view button on mobile is squashed and if you put a 700 width window on you computer, which is the perfect width on my desktop, on mobile, the document will be to wide for the screen which is really not practical
